src/include/units/bits/basic_concepts.h

```diff
@@ -202,7 +202,47 @@ concept UnitOf =
   Dimension<D> &&
   std::same_as<typename U::reference, typename dimension_unit<D>::reference>;
 
-// Quantity, QuantityPoint
+// Kind
+namespace detail {
+
+template<typename, Dimension>
+struct _kind_base;
```

Comment:
This leading `_` is to prevent confusion when writing `width::`. IDEs should suggest the base type name `_kind_base` and alias `base_kind`.